src/coreclr/jit/gentree.h

```diff
@@ -3586,7 +3586,7 @@ class SsaNumInfo final
     }
 };
 
-// Common supertype of [STORE_]LCL_VAR, [STORE_]LCL_FLD, PHI_ARG, LCL_VAR_ADDR, LCL_FLD_ADDR.
+// Common supertype of [STORE_]LCL_VAR, [STORE_]LCL_FLD, PHI_ARG, LCL_ADDR, FIELD_ADDR.
```

Comment:
```suggestion
// Common supertype of [STORE_]LCL_VAR, [STORE_]LCL_FLD, PHI_ARG, LCL_ADDR.
```

`FIELD_ADDR` does not derive from `GenTreeLclVarCommon`